keep/conditions/threshold_condition.py

```diff
@@ -10,19 +10,56 @@ class ThresholdCondition(BaseCondition):
         BaseCondition (_type_): _description_
     """
 
-    def __init__(self, condition_type, condition_config):
-        super().__init__(condition_type, condition_config)
+    def __init__(self, *kargs, **kwargs):
+        super().__init__(*kargs, **kwargs)
+        self.levels = []
 
-    def apply(self, compare_to, compare_value) -> bool:
-        """apply the condition.
+    def _check_if_multithreshold(self, compare_to):
```

Comment:
```suggestion
    def _check_if_multithreshold_and_set_levels(self, compare_to):
```
or perhaps split to 2 functions?